src/eve/codegen.py

```diff
@@ -148,7 +160,7 @@ def format_cpp_source(
         sort_includes: bool = False,
     ) -> str:
         """Format C++ source code using clang-format."""
-        args = ["clang-format"]
+        args = [str(_CLANG_FORMAT_EXECUTABLE)]
```

Comment:
This shouldn't be needed:
```suggestion
        args = [_CLANG_FORMAT_EXECUTABLE]
```